AGGREGATE.md

```diff
@@ -187,6 +187,26 @@ The `payload` will need to contain all the information needed for the aggregatio
 
 The payload should be encrypted via [HPKE](https://datatracker.ietf.org/doc/draft-irtf-cfrg-hpke/), to public keys specified by the processing origins at some well-known address  `/.well-known/aggregation-service/keys.json` that the browser can fetch. Note that we are avoiding using the `attribution-reporting` namespace because many APIs may want to use this infrastructure beyond attribution reporting.
 
+The  processing origin will respond with a set of keys and a cache control header which dictates how long to store the keys for. The browser will use these keys for the max-age as specified by the header. The scheme of the JSON encoded public keys is as follows:
+
+
+```
+{
+  "version": "<format version>",
+  "keys": [
+    {
+      "id": "<arbitrary string identifying the key>",
```

Comment:
can we specify a maximum string length here, e.g. up to 128 chars?

Reply:
Fixed. Thanks!